Danki2/Assets/Scripts/Abilities/AbilityBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Abilities;
+using Assets.Scripts.Abilities.InstantCast;
+using System;
+using System.Collections.Generic;
+
+namespace Assets.Scripts.Abilities
+{
+    public enum AbilityReference
```

Comment:
I think you're right that we could have a better name or this, I've got no idea what though haha